superset/views/core.py

```diff
@@ -1644,9 +1701,51 @@ def created_dashboards(self, user_id):
 
     @api
     @has_access_api
+    @expose('/user_slices', methods=['GET'])
+    @expose('/user_slices/<user_id>/', methods=['GET'])
+    def user_slices(self, user_id=None):
```

Comment:
Are you familiar with the `/sliceasync/api/read` endpoint? It's generated by FAB and will apply to proper filtering (same as CRUD). It has a mini language (same as CRUD) to filter rows as in `/sliceasync/api/read?_flt_0_owners=1`

Reply:
Yes, but I want to use the viz.py pipeline so I don't handle the reformatting of the data.

Reply:
I just followed the pattern for the other endpoints in this file.

Reply:
Looks like the FAB query language does not allow filtering with an or... I want filter on 'owner' or 'fav_star' so the same slice can be used in overlays by multiple users without them being owners. http://flask-appbuilder.readthedocs.io/en/latest/quickhowto.html#rest-api

Reply:
Gotcha, the star feature is a good way to not show all slices yet to allow for slices you don't own.